### DISCUSSION TITLE
[Show and Tell] Ethereum Ecosystem Faucets

### DISCUSSION THREAD

**Discussion by Maxservais** [ethereum-optimism/developers#269] [Show and tell]
### Description

Hey everyone,

Maxime here from [Ethereum Ecosystem](https://www.ethereum-ecosystem.com), an unofficial ecosystem page for Ethereum and its L2s like Base and Optimism.

I recently started to run PoW faucets built by [pk910](https://twitter.com/_pk910_) for [Ethereum](https://www.ethereum-ecosystem.com/faucets/ethereum-sepolia), [Base](https://www.ethereum-ecosystem.com/faucets/base-sepolia) and [Optimism Sepolia](https://www.ethereum-ecosystem.com/faucets/optimism-sepolia).

Would love to get your feedbacks!
Also, you can send funds you don't need anymore to replenish the faucets here:
0xd9e53C74C20EF035f16D8c6fea8aE7AefcE6945D

Cheers,
Maxime

**Comment by sbvegan** [comment]
That's awesome, thank you for your contribution :) I'll give it a try next time I need some testnet ether!
